packages/react-ui/src/widgets/WrapperWidgetUI.js

```diff
@@ -244,16 +244,18 @@ function WrapperWidgetUI(props) {
                     {option.name}
                   </MenuItem>
                 ))}
-              </Menu>
+              </PaperMenu>
             </>
           )}
-        </Grid>
-      </Grid>
+        </ActionsGrid>
+      </Header>
       {/* TODO: check collapse error */}
       <Collapse ref={wrapper} in={expanded} timeout='auto' unmountOnExit>
-        <Box className={classes.content}>{props.children}</Box>
+        <Box xs pt={1}>
```

Comment:
This xs prop is causing a warning in the console... Probably you don't need it. Please take a look.

```
Warning: Received `true` for a non-boolean attribute `xs`.

If you want to write it to the DOM, pass a string instead: xs="true" or xs={value.toString()}.
```

Reply:
The prop `xs` doesn't exist in `Box`. Remove it

Reply:
```suggestion
        <Box pt={1}>
```